src/commands/case_management.py

```diff
@@ -143,8 +148,16 @@ async def cmd_case_management_active(ctx: Context):
         return
 
     # We either have a valid case or we've left the method at this point.
-    async with ctx.bot.board.modify_rescue(rescue) as case:
+    async with ctx.bot.board.modify_rescue(rescue, impersonation=ctx.user.account) as case:
+        logger.debug(f"Switching case to active = {not case.active}")
         case.active = not case.active
+        logger.debug(f"Length: {len(ctx.words_eol)}")
+        if len(ctx.words_eol) > 2:
```

Comment:
[suggestion] in `ACTIVE_PATTERN` you will have `tokens.remainder`, i suggest checking that instead